index/server/pkg/server/endpoint.go

```diff
@@ -543,6 +543,12 @@ func buildIndexAPIResponse(c *gin.Context, indexType string, wantV1Index bool, p
 	if params.Deprecated != nil {
 		util.FilterDevfileDeprecated(&index, *params.Deprecated, wantV1Index)
 	}
+	if index == nil || len(index) == 0 {
+		c.JSON(http.StatusNotFound, gin.H{
```

Comment:
TBH, I think this should actually be `http.StatusOK`.

@michael-valdron @Jdubrick It might be unusual to have a registry return no stacks, but IMO, I think 200 is more appropriate here than 404 or 500, especially as we work towards a more dynamic registry in https://github.com/devfile/api/issues/1505

Reply:
I'm okay with reverting it back to a `200` response. @Horiodino feel free to revert that change